### DISCUSSION TITLE
stop notifications redirect to new page

### DISCUSSION THREAD

**Discussion by dilhansiriwardhana** [dailydotdev/daily#1524] [Feature Requests]
when user clicks notifications bell icon . it redirect to new page . that sucks .instead show notifications in same window . like facebook does . when it redirects it kills ux .please consider about making this come live.

**Comment by idoshamun** [comment]
Hi @dilhansiriwardhana,

Thanks for the suggestion. We know that some love our implementation and others do not appreciate it.
There's no one size fits all and we don't have plans to change the current implementation.
